src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -678,6 +705,18 @@ int sd_dhcp_lease_save(sd_dhcp_lease *lease, const char *lease_file) {
         if (r >= 0)
                 serialize_dhcp_routes(f, "ROUTES", routes, r);
 
+        r = sd_dhcp_lease_get_vendor_specific(lease, &data, &data_len);
```

Comment:
Ah, it's exported in the header file. Alright then. Just fix the `-ENOMEM` thing and we're good to go I guess. @teg?